packages/cms-cli/bin/hs-init.js

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+const { Command } = require('commander');
+
+const { initializeConfigCommand } = require('../commands/init');
+
+const program = new Command('hs init');
+initializeConfigCommand(program);
+program.parse(process.argv);
```

Comment:
We have been keeping `hscms` in sync, so you'll need to add `hscms-init.js` as well.